### ISSUE TITLE
Change Focus Area Issue While Debugging

### ISSUE THREAD

**Issue by zohaibmalik994** [microsoft/vscode#136574]
Issue Type: <b>Bug</b>

When I have multiple windows open and I have different focus areas set for each. Then when I try to debug the code and if it returns an error, it shows an error on both windows on that line and also changes the position to that error for both windows and I have re-navigate to the desired area again, however, it should only change position for the active window and not the other.
(Also, sometimes we have more than 2 windows and the same issue cause frustration to set areas back for each window separately.)
Is this something a default behavior or can it be considered as an issue?

![Issue While Debugging](https://user-images.githubusercontent.com/53822892/140614358-e12e3f05-e6a9-46af-b334-4a1d87398705.png)
(Attached is a screenshot for further clarification. I am using though Adobe ExtendScript Debugger with JavaScript code.)

VS Code version: Code 1.62.0 (b3318bc0524af3d74034b8bb8a64df0ccf35549a, 2021-11-03T15:23:01.379Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM)2 Quad CPU    Q9650  @ 3.00GHz (4 x 3000)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: unavailable_off<br>opengl: enabled_on<br>rasterization: unavailable_off<br>skia_renderer: enabled_on<br>video_decode: unavailable_off<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: unavailable_off|
|Load (avg)|undefined|
|Memory (System)|5.87GB (0.83GB free)|
|Process Argv|--crash-reporter-id ced21870-82c3-4a0c-8d37-f749c37e0e4f|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
extendscript-debug|Ado|1.1.2
xd|ado|1.2.3
texttojsxbin|mot|0.0.5

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
vsccppwtct:30382698
bridge0708:30335490
pygetstartedt3:30385195
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis400:30386382
vsaa593cf:30376535
vssld246:30386377

```

</details>

**Comment by weinand** [comment]
I agree, the exception should only be revealed in the active editor.
